data/entities.yaml

```
@@ -868,3 +868,27 @@
   description:
   - An I/O cable can be used to communicate with an adjacent robot.
   properties: [portable]
+
+- name: rubber band
+  display:
+    attr: device
+    char: 'O'
+  description:
+  - "A rubber band can tie multiple commands together so that other robots can't execute
+    commands in between them.  It can be used via the 'atomic' command. For example, suppose
+    robot A executes the following code:"
+  - |
+    b <- ishere "rock"; if b {grab} {}
+
+  - "This seems like a safe way to execute 'grab' only when there is a
+    rock to grab.  However, it is actually possible for the 'grab' to
+    fail, if some robot B tries to grab the rock at the same time:
+    first, robot A executes 'ishere' which returns true; next, robot B
+    grabs the rock; finally, robot A tries to execute 'grab', which throws
+    an exception."
```

I think this is overly detailed and something funny would be more appropriate.
```suggestion
    fail, if some other robot B snatches the rock right after robot A sensed
    it and before robot A got around to grab it on the next game tick."
  - This will make robot A very sad and it will crash.
```